### PR TITLE
Implement querying of team assignment using the practitioner Identifier

### PR DESCRIPTION
**Pull request by rehammuzzamil**
https://github.com/opensrp/opensrp-server-web/issues/936

### CODE REVIEW

File: src/main/java/org/opensrp/service/OrganizationService.java [hunk 1]
```diff
@@ -358,4 +359,15 @@ public void unassignLocationAndPlan(String planIdentifier) {
...
 		return organizationRepository.countAllOrganizations();
 	}
+
+	@PreAuthorize("hasRole('ORGANIZATION_VIEW')")
+	@PostFilter("hasPermission(filterObject, 'ORGANIZATION_VIEW')")
+	public List<Organization> getAllOrganizationsByOrganizationIds(List<Long> organizationIds) {
+		return organizationRepository.getOrganizationsByIds(organizationIds);
+	}
+
+	public List<Organization> getOrganizationsByPractitionerIdentifier(String practitionerIdentifier) {
```
Comment (dubdabasoduba) [inline src/main/java/org/opensrp/service/OrganizationService.java:369]: Please add the following to filter by roles and permissions
```
@PreAuthorize("hasRole('ORGANIZATION_VIEW')")
@PostFilter("hasPermission(filterObject, 'ORGANIZATION_VIEW')")
```

File: src/main/java/org/opensrp/service/PractitionerService.java [hunk 2]
```diff
@@ -113,6 +113,20 @@ public void deletePractitioner(String identifier) {
 		
 	}
 
+	@PreAuthorize("hasRole('PRACTITIONER_VIEW')")
```
Comment (dubdabasoduba) [inline src/main/java/org/opensrp/service/PractitionerService.java:116]: Update the permissions and roles check to the following

```
@PreAuthorize("hasRole('ORGANIZATION_VIEW')")
@PostFilter("hasPermission(filterObject, 'ORGANIZATION_VIEW')")
```

**Review by dubdabasoduba — APPROVED** [review]

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]
Other than the roles & permissions everything else looks okay to me

File: src/main/java/org/opensrp/service/OrganizationService.java (hunk 1 above)
Comment (rehammuzzamil) [reply src/main/java/org/opensrp/service/OrganizationService.java:371]: Added

File: src/main/java/org/opensrp/service/PractitionerService.java (hunk 2 above)
Comment (rehammuzzamil) [reply src/main/java/org/opensrp/service/PractitionerService.java]: Done

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**